packages/velo-external-db-core/lib/service/schema.js

```diff
@@ -40,6 +44,15 @@ class SchemaService {
         await this.schemaInformation.refresh()
         return {}
     }
+
+    prepareDbsList(dbs, schemaSupportedOperations) {
```

Comment:
why have prepare db list here and prepare fields on commons ? and let's take care of the name here as well.